src/components/Forms/Input.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+const Input = React.forwardRef(
+  (
+    {
+      type,
+      name,
+      disabled = false,
+      errors,
+      errorMessage,
+      children,
+      onChange,
+      ...props
+    },
+    ref
+  ) => {
+    const [active, setActive] = React.useState(false);
+
+    function handleActivation(e) {
+      setActive(!!e.target.value);
+      onChange(e);
+    }
+
+    return (
+      <>
+        <div
+          className={[
+            'relative border-2 rounded-md mb-2 text-background border-tertiary',
+            errors ? 'border-red-500' : '',
+          ].join(' ')}
+        >
+          <input
+            className={[
+              'outline-none w-full rounded bg-transparent text-sm transition-all duration-200 ease-in-out p-2',
+              active ? 'pt-6' : '',
+              errors ? 'error' : '',
+            ].join(' ')}
+            name={name}
```

Comment:
Give the input an id so whenever you click on the label that you used as  placeholder it will active the input